packages/cfd/.eslintrc.js

```diff
@@ -7,4 +7,7 @@ module.exports = {
             webpack: { config: webpackConfig({}) },
         },
     },
+    rules: {
+        'no-underscore-dangle': 0,
```

Comment:
this is needed to turn the eslint 'no-underscore-dangle' warning for identifiers off because we are using dangling underscores for identifiers e.g. in cfd-personal-details-form.tsx at line 272

Reply:
any reason why we use dangling variable name? 
You can move this to another PR to refactor the component instead of this PR.

Reply:
I've renamed the variable and removed this rule 👍🏻